docs/server-side/v2/how-to-guides/live-transcription-hls.mdx

```diff
@@ -0,0 +1,184 @@
+100ms is excited to offer auto-generated live transcription in English.
+
+![create template](/docs/v2/live-transcription-workflow.png)
+
+
+### Enabling and Configuring Live Transcription
+
+
+<Tabs id="implementation" items={['Dashboard', 'Live Stream API']}/>
+<Tab id='implementation-0'>
+
+
+#### Creating a new template
+1. On the 100ms Dashboard, click on ‘Create Template’.
+
+![create template](/docs/v2/create-a-template.png)
+
+2. Select ‘Live Streaming’ as your use-case and click on ‘Configure’.
+
+![select live streaming](/docs/v2/rtmp-ingestion/rtmp-in-create-template-1.png)
+
+3. Configure the template by answering the set of questions on the first screen and click 'Select Add-ons'.
+
+4. Select 'Yes' to the question 'Do you require live transcription for your live streams?'.
+![select yes to live transcription](/docs/v2/create-template-live-transcription-question.png)
+
+5. Deploy the template and proceed further. On the final screen, use the different roles available to experience the 
+
+Read up more on the live streaming experience [here](/get-started/v2/get-started/features/live-streaming).
+
+
+
+#### Editing an existing template
+1. Go to the template configuration
+    
+2. Click on ‘Transcription (Beta)’ tab.
+
+3. Enable ‘Live Transcription’ toggle.
+
+![enable live transcription](/docs/v2/live-transcription-enable-toggle.png)
+
+</Tab>
+
+
+<Tab id='implementation-1'>
+You can refer to our [Live Streaming API documentation](/server-side/v2/api-reference/live-streams/start-live-stream-for-room) to start recording.
+
+1. You are required to mark the transcription `enabled` flag in the request as `true` and select `live` in `modes`to transcribe unless you’ve already enabled transcription for the given room template. Refer Dashboard.
+2. You can override the live transcription settings given in the template when starting the live stream through the Live Stream API.
+3. If no input is given, the API will pick up the transcription configuration from the template policy / configuration.
+
+
+</Tab>
+
+
+
+#### Advanced Configuration
+
+There are two advanced configuration that are currently offered.
+
+1. **Custom Vocabulary**: Add non-dictionary words like names, abbreviations, slang, and technical jargon which may not be recognised by the AI model for better transcription accuracy.
+2. **Language:** Select the language that needs to be transcribed. This will be the spoken language and not for the purpose of translation. Transcription only supports **English** language at this point of time. Support for more languages will come soon.
+
+![advanced transcription config](/docs/v2/advanced-transcription-config.png)
+
+### Example Output
+
+#### Master manifest (master.m3u8)
+```
+#EXTM3U
+#EXT-X-VERSION:6
+#EXT-X-MEDIA:TYPE=SUBTITLES,GROUP-ID="subs",LANGUAGE="en-US",NAME="English CC (auto)",AUTOSELECT=YES,DEFAULT=YES,URI="subtitles/subtitles.m3u8",FORCED=YES,CHARACTERISTICS="public.accessibility.transcribes-spoken-dialog"
```

Comment:
is this the latest snipped? I think we removed the forced=yes from this tag. 
cc @piyush1104

Reply:
yeah, let me fix this one.

Reply:
Resolving this.